server/events/templates/diverged.tmpl

```diff
@@ -1,5 +1,5 @@
 {{ define "diverged" -}}
```

Comment:
This file needs to be renamed if you are going to be consistent with diverged refactor